Source/Microsoft.Teams.Apps.CompanyCommunicator.Send.Func/CompanyCommunicatorSendFunction.cs

```diff
@@ -115,16 +132,27 @@ public class CompanyCommunicatorSendFunction
                         incomingUserDataEntity.AadId)
                     : Task.FromResult<UserDataEntity>(null);
 
-                await Task.WhenAll(getActiveNotificationEntityTask, getUserDataEntityTask);
+                await Task.WhenAll(getActiveNotificationEntityTask, getGlobalSendingNotificationDataEntityTask, getUserDataEntityTask);
 
                 var activeNotificationEntity = getActiveNotificationEntityTask.Result;
 
+                var globalSendingNotificationDataEntity = getGlobalSendingNotificationDataEntityTask.Result;
```

Comment:
Nit: you can use `await` again here, prevents removal of `.WhenAll` causing sync locks or deadlocks. The task memoizes the result.